lib/utils/embeddings/setup/schema.ts

```diff
@@ -0,0 +1,113 @@
+export const schema = {
```

Comment:
Changed this to a static import for the benefits of TS (eg. can access `typeof schema.properties[0].name`)